### ISSUE TITLE
Feature Request : Simple Image Tweaking / Filter

### ISSUE THREAD

**Issue by rasyidf** [SkalskiP/make-sense#207]
**Is your feature request related to a problem? Please describe.**
Would it be better if we can change the brightness, gamma, contrast, or just make it grayscale to  manipulate images. make it session / project based, that can persist when selected image changed.

**Describe the solution you'd like**
Something like drawer or panel that can manipulate base image tweaking. Brightness, Contrast, Shadow or Highlight.

it would be great if we can preprocess the image before annotating, that'd be another feature request

**Describe alternatives you've considered**

**Additional context**
![image](https://user-images.githubusercontent.com/28984914/138466623-786ebd8e-13c9-4c09-9420-888a8913a5d2.png)

PS: Sorry, i used your profile

**Comment by SkalskiP** [comment]
Hello, again @rasyidf 👋  This is such a good idea 😮 ! It is such a cool feature and something that we could be proud to have. However, we need to plan it in more detail. Questions that immediately come to my mind:

1. Do we want to preserve the result of that operation or is it just temporal to make it easier to label? If we want to preserve results this task would be dependant on #169 as we, first of all, need to have a way to save and load not only labels but the whole project. I myself think those operations should be temporal, especially at the beginning to keep things simple and not overcomplicate implementation.
2. Do we want to allow users to combine transformations? Can a single image have changed brightness and gamma at the same time?
3. Do we want to allow users to input the level of transformation via some scroll or number input? 
4. Do we want to allow users for per image transformations? 
5. Where those processing tools should be located in the UI? I think they should be easily accessable so I think that they can be placed in the left or right sidebar. Those bars are designed in a way that they can have multiple tabs inside a single sidebar. For now, the left sidebar contains only the images tab and the right only labels tab, but we can add new tabs quite easly.

**Comment by SkalskiP** [comment]
@rasyidf we should also think about task prioritization. We have lots of new ideas, and that is great! But we need to focus and decide which of them are most crucial and have the biggest impact and start with those. I was thinking maybe we can use https://github.com/SkalskiP/make-sense/discussions to do that?

**Comment by rasyidf** [comment]
you're right, we should prioritize more on completing beta phases. and discussion board would be a great idea. let's discuss more there.

**Comment by arronsullivan** [comment]
This would be very helpful.  Specifically just allowing basic min/max contrast settings. Just to make it easier to label, not to be retained.
Fiji/ImageJ has a nice 'auto' contrast where it sets the max such that 5% of pixels are saturated I believe, a very nice shortcut to see dim images.
Another 'advanced' setting would be to apply a user set min/max contrast to all images as they are opened/viewed for speed.

[Someone has implemented this into labelImg](https://www.youtube.com/watch?v=A5qwvtiSmMc&lc=UgxvF3EDuc9Z7lLK0Jt4AaABAg) but has not shared it anywhere that I can see.

Thank you, make-sense looks very nice
